modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -1087,14 +1082,15 @@ RETURN d.weeks, d.days, d.daysOfWeek
 3+d|Rows: 1
 |===
 
+`d.weeks` has a value of 3 because the 25 days from the query are three full weeks (or 21 days). `d.daysOfWeek` are the remaining days, counting towards the next full week.
+See tables _Most significant components_ and _Less significant components in <<cypher-temporal-accessing-components-durations>>.
+
 ======
 
 
-.+duration+
+.Get the most significant second-based components of a `DURATION` value
```

Comment:
```suggestion
.Get the most significant seconds-based components of a `DURATION` value
```
Since you use _days-based_ earlier, this should thus be _seconds-based_.